cms/djangoapps/contentstore/views/item.py

```diff
@@ -266,6 +269,8 @@ def service(self, block, service_name):
                 return DjangoXBlockUserService(self._user)
             if service_name == "studio_user_permissions":
                 return StudioPermissionsService(self._user)
+            if service_name == "lti-configuration":
+                return ConfigurationService(LTIConsumerHideFieldsFlag)
```

Comment:
@muzaffaryousaf You don't need to do anything further other than have the LTI xblock import django settings.  You can have it conditionally import the settings (with a try-catch around it like [here](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/video_module/video_module.py#L77)) if we are concerned about it being used outside of django.  I would rather you make that change than to add LTI dependency in this xblock runtime.